portal/src/util/escapeURI.ts

```diff
@@ -0,0 +1,3 @@
+export function encodeURIComponentRFC3986(str: string): string {
```

Comment:
Just `encodeURIComponent` is enough practically.

Reply:
updated to using encodeURIComponent only @kiootic